python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -494,13 +480,32 @@ def update_operator_states(topology: Topology) -> None:
             op.mark_execution_completed()
 
 
+@dataclass
+class OpSchedulingStatus:
+    """The scheduling status of an operator."""
+
+    # Whether the operator is runnable.
```

Comment:
what exactly does "runnable" mean here? can we expand on this definition?
also, wondering if it would make sense to connect this with `OpState`, or combine the two classes together

Reply:
`select_op_to_run` checks which ops are runnable, and choose the best op to run. "runnable" means this. Will add a comment. 
also good suggestion to incorporate this class with `OpState`.